### PR TITLE
[hotfix] add and fix marvel pruning script

### PR DESCRIPTION
**Pull request by emanuil-tolev**
https://github.com/DOAJ/doajpm/issues/1525

### CODE REVIEW

File: doajtest/unit/test_prune_marvel.py [hunk 1]
```diff
@@ -0,0 +1,38 @@
+import datetime, calendar
+from dateutil.relativedelta import relativedelta
+from freezegun import freeze_time
+
+from doajtest.helpers import DoajTestCase
+from portality.scripts.prune_marvel import generate_delete_pattern
+
+
+class TestFeed(DoajTestCase):
```
Comment (Steven-Eardley) [inline doajtest/unit/test_prune_marvel.py]: Is this classname as intended?

File: portality/scripts/prune_marvel.py [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+    delete_date = now - relativedelta(months=3)
+
+    return ".marvel-{0}*".format(delete_date.strftime('%Y.%m'))
+
+
+def main():
+    delete_pattern = generate_delete_pattern()
+    print "To delete: {0}".format(delete_pattern)
+
+    if len(delete_pattern) < 8:
```
Comment (Steven-Eardley) [inline portality/scripts/prune_marvel.py]: Does this mean that the delete pattern is shorter than the string ```.marvel-*```? That's 9 characters. Or have I missed the point of this?

Comment (emanuil-tolev) [reply portality/scripts/prune_marvel.py]: That is the idea, yes. I think this kind of check should be done in the tests, now that we have tests to verify the behaviour of the code. In fact we know what the length of the output must be: 16. I've documented that with a new unit test assertion.

Comment (emanuil-tolev) [reply portality/scripts/prune_marvel.py]: (and removed this runtime exception)

File: doajtest/unit/test_prune_marvel.py (hunk 1 above)
Comment (emanuil-tolev) [reply doajtest/unit/test_prune_marvel.py]: Yes, I have decided to name all my tests that!
(fixed ta)

**Pull request closed without merging**